packages/ozone/src/lexicon/lexicons.ts

```diff
@@ -11127,7 +11145,7 @@ export const schemaDict = {
         type: 'object',
         description:
           'Logs account status related events on a repo subject. Normally captured by automod from the firehose and emitted to ozone for historical tracking.',
-        required: ['timestamp', 'status', 'active'],
+        required: ['timestamp', 'active'],
```

Comment:
TODO: if this PR doesn't make it into the main branch, we'd wanna port this change over to the parent PR.